pyscriptjs/src/plugin.ts

```diff
@@ -57,19 +61,30 @@ export class Plugin {
 
 export class PluginManager {
     _plugins: Plugin[];
+    _pythonPlugins: any[];
```

Comment:
Since python plugins have the same lifecycle, we should perhaps create a type/interface for these and use it as type here instead of any

Reply:
Yeah, we could. I didn't want to spend too much time on typing right now because, effectively, the objects being added here are actually PyProxy objects. This may change later though. We'll see